frontend/src/metabase/browse/databases/BrowseDatabases.tsx

```diff
@@ -52,27 +66,75 @@ export const BrowseDatabases = () => {
       <BrowseDataHeader />
       <BrowseMain>
         <BrowseSection>
-          <DatabaseGrid data-testid="database-browser">
-            {databases.map(database => (
-              <div key={database.id}>
-                <DatabaseCardLink to={Urls.browseDatabase(database)}>
-                  <DatabaseCard>
-                    <Icon
-                      name="database"
-                      color={color("accent2")}
-                      className={CS.mb3}
-                      size={32}
-                    />
-                    <Title order={2} size="1rem" lh="1rem" color="inherit">
+          <BrowseGrid data-testid="database-browser">
+            {databases &&
+              databases.length > 0 &&
+              databases.map(database => (
+                <Link to={Urls.browseDatabase(database)} key={database.id}>
+                  <DatabaseCard
+                    bg="bg-white"
+                    className={cx(CS.rounded, CS.bordered, DB.dbCard)}
+                  >
+                    <Icon name="database" color={color("accent2")} size={32} />
+                    <Title order={2} size="md" lh="md" color="inherit">
                       {database.name}
                     </Title>
                   </DatabaseCard>
-                </DatabaseCardLink>
-              </div>
-            ))}
-          </DatabaseGrid>
+                </Link>
+              ))}
+
+            {isAdmin && (
+              <Link to={newDatabase()} onClick={() => trackAddDatabaseDBList()}>
+                <DatabaseCard
+                  className={cx(CS.rounded, CS.bordered, DB.addCard)}
+                >
+                  <Group spacing="xs">
+                    <CardImageWrapper database={"postgres"} />
+                    <CardImageWrapper database={"mysql"} />
+                    <CardImageWrapper database={"snowflake"} />
+                  </Group>
+                  <div>
+                    <Title order={2} size="md" lh={1} color="inherit">
+                      {t`Add a database`}
+                    </Title>
+                    <Text
+                      color="inherit"
+                      fz="sm"
+                      lh={1}
+                      span
+                    >{t`20+ data connectors. Start exploring in minutes.`}</Text>
+                  </div>
+                </DatabaseCard>
+              </Link>
+            )}
+          </BrowseGrid>
         </BrowseSection>
       </BrowseMain>
     </BrowseContainer>
   );
 };
+
+const CardImageWrapper = ({ database }: { database: string }) => {
+  return (
+    <Box
+      bg="white"
+      h="xl"
+      w="xl"
+      className={CS.rounded}
+      style={{
+        boxShadow:
+          "0px 0px 0px 1px rgba(0, 0, 0, 0.05), 0px 1px 4px 0px rgba(0, 0, 0, 0.10)",
```

Comment:
I am aware that we have a predefined color value called `shadow: rgba(0, 0, 0, 0.08)`.
I tried to use it here, but it looks much different than the provided design. Hence, I decided to go with the literal value for the box-shadow that was provided by the designer.

Btw, we might want to consider adding a few shadow variants for greater flexibility in the future?

Reply:
this is always worth a quick chat with the designer - this isn't some crazy unique component - either it should use shadows from the design system or, as you suggest, the design system should maybe have another variant.

Also, I would prefer doing this shadow in CSS - maybe even in that CS module like `CS.BoxShadowSomething`